.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: SOS-github-actions
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  SOS_INSTALL_DIR: ${{ github.workspace }}/install/sos
+  LIBFABRIC_INSTALL_DIR: ${{ github.workspace }}/install/libfabric
+  UCX_INSTALL_DIR: ${{ github.workspace }}/install/ucx
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - config_name: libfabric-1.7.x
```

Comment:
Any particular reason for this version?  I've been using `v1.12.x` lately... it might make sense to stay as recent as possible.

Reply:
Oh nevermind... I see we use `v1.7.x` in Travis CI.  This is fine for now... we can update that later.